client/vite.config.ts

```diff
@@ -30,11 +30,16 @@ export default defineConfig({
         display: 'standalone',
         background_color: '#ffffff',
         icons: [
+          {
+            src: 'images/CSF_feet_144_sq_px.png',
+            sizes: '144x144',
+            type: 'image/png'
+          },
           {
             src: 'images/CSF_feet_144_sq_px.png',
             sizes: '144x144',
             type: 'image/png',
-            purpose: 'maskable'
+            purpose: 'any'
```

Comment:
Don't we just need the first one to have either no purpose or purpose 'any' and then the following ones to have purpose 'maskable'?